colour/colour.go

```diff
@@ -136,3 +136,34 @@ func Yellow(message string) string {
 func Grey(message string) string {
 	return Bright + FgBlack + message + Reset
 }
+
+// StripAllColourCodes strips all the ANSI colour codes from a string
+func StripAllColourCodes(message string) string {
```

Comment:
I wish there was a way of just iterating through all the constants, rather than having to keep 2 lists in sync (what happens when a new constant is added without updating this function)... but I can't see a way to do that with the way `constants.go` is currently laid out